### DISCUSSION TITLE
Pourquoi un PIC32 ?

### DISCUSSION THREAD

**Discussion by zuzu59** [philouxy/TOTEM_FIXME#14] [Ideas]
Hello,
Je viens de lire ce joli projet pour FIXME 👌
Ma toute 1ère question est, mais pourquoi donc utiliser un vieux PIC32 à l'époque de l'iot ?
Un esp32-c3 serait beaucoup plus adapté pour ce projet 😉
Non seulement il a beaucoup plus de mémoire, mais est nettement plus puissant. De plus on pourrait faire de l'OTA pour la maintenance 💪
Et ça ce programme exactement de la même manière qu'un PIC32 avec Arduino ou code ! 
Merci pour la réponse. 
Cordialement 
Ch. Zufferey (zuzu) 

Ps: pourquoi ne pas avoir ouvert les discussions sur le fork de fixement ?

**Comment by philouxy** [comment]
Hello Zuzu, 

C'était un projet que j'avais proposé à l'ETML-ES (2017) - pour qu'on ait une base de totem à Fixme - la seule contrainte pour un projet de semestre à l'ES c'est que les étudiants puissent développer un Firmware sur PIC32, car c'est le uC  qu'ils voient en détails en 2ème année de formation de technicien. 

Effectivement le but, et de refaire une nouvelle version soit basé sur un STM32 ou directement avec un ESP32... à voir (on peut même faire deux versions)

Actuellement, je revois la partie technique (hardware) <=> partie alimentation / panneau solaire / batterie... J'ai aussi recommandé des panneaux de leds (8 x 32). 

La première itération du totem est à l'ES, mais je pourrais l'amener à Fixme, y a plusieurs choses a repensez tant au niveau hardware (électronique / mécanique) que sur le partie partie Firmware, sans oublier un partie software (application)

Bref y a encore du job, et toutes les idées, petits mains qui veulent se greffer au projet son les bienvenues. 

Je vais regarder pourquoi le module discussion n'est pas activé sur le fork de Fixme ;) 

Amitié - Philou (Fixme members)
